File: data/src/main/java/com/blackmirror/dongda/data/model/response/EnrolResponseBean.java
package com.blackmirror.dongda.data.model.response;

import java.io.Serializable;

/**
 * Create By Ruge at 2018-05-15
 */
public class EnrolResponseBean extends BaseResponseBean {


    /**
     * result : {"recruit_id":"5afce8db4946b50d15a2acd0"}
     */

    public ResultBean result;

    public static class ResultBean implements Serializable {
        /**
         * recruit_id : 5afce8db4946b50d15a2acd0
         */

        public String recruit_id;
    }
}

File: data/src/main/java/com/blackmirror/dongda/data/model/response/EnrolResponseBean.kt
package com.blackmirror.dongda.data.model.response

import java.io.Serializable

/**
 * Create By Ruge at 2018-05-15
 */
class EnrolResponseBean : BaseResponseBean() {


    /**
     * result : {"recruit_id":"5afce8db4946b50d15a2acd0"}
     */

    var result: ResultBean? = null

    class ResultBean : Serializable {
        /**
         * recruit_id : 5afce8db4946b50d15a2acd0
         */

        var recruit_id: String? = null
    }
}
